shared/output/gapic/templates/googleads/google-ads-googleads.gemspec

```diff
@@ -1,7 +1,7 @@
 # -*- ruby -*-
 # encoding: utf-8
 
-require File.expand_path("lib/google/ads/google_ads/version", __dir__)
+require File.expand_path("lib/google/ads/googleads/version", __dir__)
```

Comment:
The Google Ads file paths have been changed because of he configured inflections.